browser/ui/webui/sources.gni

```diff
@@ -3,13 +3,15 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import("//brave/components/brave_education/buildflags.gni")
 import("//brave/components/brave_vpn/common/buildflags/buildflags.gni")
 import("//brave/components/playlist/common/buildflags/buildflags.gni")
 
 # This is required for chromium_src overrides.
 brave_browser_ui_webui_configs_deps = [
   "//base",
   "//brave/browser/ui",
+  "//brave/components/brave_education:buildflags",
```

Comment:
wait. 
1. please move `buildflags` target into `//brave/components/brave_education/buildflags/BUILD.gn`
2. please add `assert(enable_brave_education)` into  `//brave/components/brave_education/BUILD.gn`

Reply:
actually, you can make this in a follow up. Let's merge this PR to fix occasional build failures.